README.md

```diff
@@ -18,13 +18,36 @@ KubeVirt Ansible consists of a set of Ansible playbooks that deploy fully functi
 
 ## Usage
 
+### Deploy
 To deploy KubeVirt on an existing OpenShift cluster run the command below. For more information on clusters and other deployment scenarious see [playbooks instructions](./playbooks/README.md).
 
 ```
 ansible-playbook -i localhost playbooks/kubevirt.yml -e@vars/all.yml
 ```
 >**Note:** Check default variables in [vars/all.yml](./vars/all.yml) and update them if needed.
 
+### E2E Testing
+
+After a successful deployment or using an exist cluster, make sure you can `oc login` into cluster.
+
+```
+make generate-tests
+```
+Will compile tests from tests inside docker container and copy it to _out directory inside kubevirt-ansible repository.
+
+```
+make test
+```
+Will run all the e2e tests with kubeconfig from `~/.kube/config`, or you can pass it to tests via:
+```
+./_out/tests/<name>.test -kubeconfig=your_kubeconfig -tag=kubevirt_images_tag -prefix=kubevirt -test.timeout 60m
+```
+
+If you want to test PVC's `storage.import.endpoint` with different/internal images, we provided an ENV called `STREAM_IMAGE_URL` which you can set up in your own environment like:
```

Comment:
avoid WE in docs:

To test PVC's `storage.import.endpoint` with other images, use the `STREAM_IMAGE_URL` environment variable: